src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
```

Comment:
You can use selector `*` and don't repeat the style in the 28th line
[How selector `*` works](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors)
but you can add in it only margins or/and box-sizing